### DISCUSSION TITLE
Valheim Server not Starting

### DISCUSSION THREAD

**Discussion by broemp** [GameServerManagers/LinuxGSM#3238] [General]
Hi Everybody,

Everytime I try to start my Valheim Server it stops at DungeonDB Start. I tried different ways to install it, but it stops at DungeonDB everytime.

My Console and details
https://pastebin.com/Rw8kry6y

**Comment by dgibbs64** [comment]
This is not an issue with LinuxGSM. However it looks like your server is under a heavy load. So that might not be helping
